geostore/check_files_checksums/task.py

```diff
@@ -18,6 +18,7 @@ def parse_arguments() -> Namespace:
     argument_parser = ArgumentParser()
     argument_parser.add_argument("--dataset-id", required=True)
     argument_parser.add_argument("--version-id", required=True)
```

Comment:
As discussed, this argument can probably go now.

Reply:
This is still required to build the hash key for querying the processing assets table.